darwin/exporter/formats/nifti.py

```diff
@@ -83,7 +83,9 @@ def export(
             medical_metadata = video_annotation.slots[0].metadata
             legacy = not medical_metadata.get("handler") == "MONAI"  # type: ignore
             plane_map = medical_metadata.get("plane_map", {slot_name: "AXIAL"})
-            primary_plane = plane_map.get(slot_name, "AXIAL")
+            primary_plane = medical_metadata.get(
+                "primary_plane", plane_map.get(slot_name, "AXIAL")
```

Comment:
Plane map show which plane represents specific slot, right?
Is it correct at all to use it as fallback for primary plane?
Maybe just default to the most commonly used here?

Reply:
Oh, the idea is that annotations can be created only in the primary plane ...
When is it not the case?
Nevertheless, let's leave it as is.